RELEASE.rst

```diff
@@ -0,0 +1,36 @@
+The SunPy project is very happy to announce the release of a new package "ndcube".
+
+ndcube is a package built to represent a wide variety of data, of any number of
```

Comment:
How about "ndcube is a package built for handling, inspecting and visualizing a wide variety of data, of any number of dimensions, along with coordinate information described by FITS-WCS."